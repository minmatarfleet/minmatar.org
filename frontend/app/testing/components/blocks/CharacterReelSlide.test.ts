import { experimental_AstroContainer as AstroContainer } from "astro/container";
import { expect, test, vi } from "vitest";
import CharacterReelSlide from "@components/blocks/CharacterReelSlide.astro";

test("CharacterReelSlide defaults", async () => {
  const container = await AstroContainer.create();
  const result = await container.renderToString(CharacterReelSlide, {
    slots: {
      default: "slot mock",
    },
  });

  expect(result).toMatchSnapshot();
});
