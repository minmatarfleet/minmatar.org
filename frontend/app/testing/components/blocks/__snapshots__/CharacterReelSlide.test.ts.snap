// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CharacterReelSlide defaults 1`] = `"<div class="[ swiper-slide ]" style="margin-right: undefinedpx" data-astro-cid-pvvgnag4> slot mock </div> "`;
